packages/dropdownable/components/DropdownContents.tsx

```diff
@@ -1,26 +1,31 @@
 import React from "react";
 import wrapWithClickOutside from "react-click-outside";
```

Comment:
@mstruebing pointed out that react-click-outside is pretty outdated with the [last publish happening 5 years ago](https://www.npmjs.com/package/react-click-outside). 

We'd be open to another option here to improve this component such as this one - https://github.com/zhaluza/react-detect-click-outside

What are your thoughts on implementing this?